src/cryptography/hazmat/primitives/asymmetric/rsa.py

```diff
@@ -222,6 +223,9 @@ def rsa_recover_prime_factors(n: int, e: int, d: int) -> tuple[int, int]:
     Compute factors p and q from the private exponent d. We assume that n has
     no more than two factors. This function is adapted from code in PyCrypto.
     """
+    # reject invalid values early
+    if 17 != pow(17, e * d, n):
```

Comment:
Why 17?

Reply:
It's arbitrary, it can be pretty much any value, just needs to be < n.